pkg/cosign/keys_test.go

```diff
@@ -48,7 +129,65 @@ func TestLoadECDSAPrivateKey(t *testing.T) {
 	if _, err := rand.Read(buf[:]); err != nil {
 		t.Fatal(err)
 	}
-	if _, err := LoadECDSAPrivateKey(buf[:], []byte("wrong")); err == nil {
+	if _, err := LoadPrivateKey(buf[:], []byte("wrong")); err == nil {
 		t.Error("expected error decrypting key!")
 	}
 }
+
+func TestImportPrivateKey(t *testing.T) {
+	err := os.WriteFile("validrsa.key", []byte(validrsa), 0600)
```

Comment:
Probably best to use a temp dir , see https://github.com/sigstore/cosign/blob/0ebe3b5869e48cc411015c14f46aa925b94fba2b/cmd/cosign/cli/publickey/public_key_test.go#L47-L48

Reply:
updated